api/specs/webserver/openapi-projects.yaml

```diff
@@ -12,6 +12,12 @@ paths:
             type: string
             default: "all"
             enum: [template, user, all]
+        - name: show_hidden
+          in: query
+          schema:
+            type: boolean
+            default: false
+          description: includes projects marked as hidden in the listing
```

Comment:
not exclusively. 

The webserver API gives the option to create projects and mark them as "hidden"
The api-server service (independently) decides to create projects and mark them as "hidden". But in future version this might change...

I do not see a reason why the webserver API should document some design decision taken in a different service.